apps/ledger-live-mobile/fastlane/Fastfile

```diff
@@ -109,7 +109,7 @@ platform :ios do
 
     build_number = latest_testflight_build_number(
       version: trim_version_number(package["version"]),
-      app_identifier: MY_APP_BUNDLE_ID
+      app_identifier: "com.ledger.live"
```

Comment:
Why is this here?

Reply:
it was a fixed to be able to build iOS that Valentin told me to try -> as this PR won't be merged, i did not put any comment on it. But Valentin merged this fix on develop yesterday ;)